flink-libraries/flink-cep-scala/pom.xml

```diff
@@ -93,6 +93,8 @@ under the License.
     </dependencies>
 
     <build>
+        <!-- Not strictly necessary because of the build-helper-maven-plugin below. -->
+        <sourceDirectory>src/main/scala</sourceDirectory>
```

Comment:
the other modules containing scala code get by without this; it seems safe to remove it.